flash/data/data_utils.py

```diff
@@ -3,6 +3,7 @@
 import pandas as pd
 
 
+
 def labels_from_categorical_csv(
```

Comment:
what is this called categorical_csv didn't we fix this

Reply:
can someone give me write access on the branch it's very frustrating to have to submit pull requests from a fork every time